packages/backend/src/tests/propertyBased/arbitraries/arbitraryQuery.ts

```diff
@@ -0,0 +1,38 @@
+import { fc } from '@fast-check/vitest';
+import { getTableNames, Schema, Cardinality } from '@synthql/queries';
+import { AnyQuery } from '../../../types';
+import { arbitraryLimit } from './arbitraryLimit';
+import { arbitraryCardinality } from './arbitraryCardinality';
+import { AllTablesRowsMap } from '../properties/executeQuery';
+import { arbitraryWhere } from './rbitraryWhere';
+import { arbitraryTableName } from './arbitraryTableName';
+import { arbitrarySelect } from './arbitrarySelect';
+
+interface ArbitraryQuery<DB> {
+    schema: Schema<DB>;
+    allTablesRowsMap: AllTablesRowsMap;
+    cardinality: Cardinality;
+    validWhere: boolean;
+}
+
+export function generateArbitraryQuery<DB>({
```

Comment:
rename to `arbitraryQuery`: pay attention to the little things @jimezesinachi. They don't matter by themselves but they are a good measure to see how well you're paying attention to the code that you're writing